src/lib/mocha.ts

```diff
@@ -28,8 +28,11 @@ function createMochaInstance(
 		// Call with true tell mocha that we will be handling the file load outselves
 		this.lazyLoadFiles(true);
 
+		const isWin = path.delimiter === ';';
+
 		for (let file of files) {
-			file = path.resolve(file);
+			if (isWin) path.posix.resolve(file);
+			else file = path.resolve(file);
```

Comment:
Remove if-condition and just use posix.resolve.

I believe this is doing "if windows, use posix.resolve, else (POSIX detected), use inferred posix.resolve"